src/Cli/dotnet/commands/dotnet-complete/Complete.cs

```diff
@@ -17,63 +15,58 @@ namespace Microsoft.DotNet.Cli
 {
     internal static class Complete
     {
+        private static CompletionItem ToCompletionItem(string s) => new CompletionItem(s);
 
-        private static CompletionItem ToCompletionItem (string s) => new CompletionItem(s);
-
-        public static Func<CompletionContext, IEnumerable<CompletionItem>> TargetFrameworksFromProjectFile =>
-            (_context) =>
+        public static IEnumerable<CompletionItem> TargetFrameworksFromProjectFile(CompletionContext _)
```

Comment:
these properties was allocating memory each time they were called. We can just define static methods rather than properties that return lambdas.